krux_marathon_api/marathonapi.py

```diff
@@ -71,8 +52,12 @@ def connect(self, address, port):
             s.close()
```

Comment:
Sorry, I was referring to passing strings as parameters and not using string substitution with `%`. You should keep the exception message as is. I was just being lazy.